scripts/export.py

```diff
@@ -20,6 +20,9 @@
 OUTLIER_TYPE_FIELD = "USER27"
 RUN_ID_FIELD = "USER28"
 
+IS_OUTLIER_INDICATOR = "N: Non arms-length"
+NON_OUTLIER_INDICATOR = "Y: Is arms-length"
+
```

Comment:
I feel like this has the potential to become _incredibly_ confusing. Let's all align on the correct indicator meaning/direction this afternoon and keep this PR open until then.